packages/react-devtools-shared/src/hydration.js

```diff
@@ -325,7 +339,6 @@ export function dehydrate(
       return {
         type,
       };
-
```

Comment:
nit: Keep the empty line on 298 and 328 for consistency with the rest of the file :wink:

Reply:
:-D  Reverted